### DISCUSSION TITLE
pptx format ignores column width setting

### DISCUSSION THREAD

**Discussion by yonisidi** [quarto-dev/quarto-cli#10054] [Q&A] [answered]
### Description

Columns do not react to width setting [pptx column documentation](https://quarto.org/docs/presentations/powerpoint.html#multiple-columns)

````qmd
---
title: "Slides Title"
author: "Author Name"
date: today
format: pptx
---

## My Header

::: columns
::: {.column  width="10%"}
- bullet 1
- bullet 2
:::

::: {.column width="90%"}
![An image](https://placehold.co/600x400.png){#fig-myfig}
:::
:::


````

**Comment by mcanouil** [comment, marked as answer]
See:

- https://github.com/quarto-dev/quarto-cli/issues/2635

**Comment by yonisidi** [comment]
Is there a way to force a slide layout on pandoc? I could make preset width layouts and reference them for specific slides.

**Reply by mcanouil** [reply]
You can't have custom layout.
Only the one recognise by Pandoc can be used thus the need to create the template out of Pandoc's template.
